### PR TITLE
contrib/dmidecode: new package (3.5)

### PR DESCRIPTION
**Pull request by nekopsykose**

### CODE REVIEW

File: contrib/dmidecode/template.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+pkgname = "dmidecode"
+pkgver = "3.5"
+pkgrel = 0
```
Comment (q66) [inline contrib/dmidecode/template.py:3]: i believe this should be `archs = ["x86_64"]`, no?

Comment (nekopsykose) [reply contrib/dmidecode/template.py:3]: aarch64 also has smbios/dmi afaik. so perhaps just those 2, not sure if riscv will ever support that either..

Comment (nekopsykose) [reply contrib/dmidecode/template.py:3]: hm, think riscv also has it

Comment (nekopsykose) [reply contrib/dmidecode/template.py:3]: from what i can tell any platform that implements EFI can optionally then provide the smbios tables for this to end up working, i guess

Comment (q66) [reply contrib/dmidecode/template.py:3]: these 3 then

**Pull request merged**